api/v6/report_server.thrift

```diff
@@ -156,7 +157,8 @@ struct ReportData {
   9: i64              column,          // column number of the report main section (not part of the path).
   10: Severity        severity,        // Checker severity.
   11: ReviewData      reviewData,      // Bug review status information.
-  12: DetectionStatus detectionStatus  // State of the bug (see the enum constant values).
+  12: DetectionStatus detectionStatus, // State of the bug (see the enum constant values).
+  13: i64             bugPathLength,   // Length of the bug path.
```

Comment:
What is the policy on commas after the last item?

Reply:
If we add a new item to this type, we don't have to modify the last item to place a comma at the end.